packages/ckeditor5-engine/src/index.ts

```diff
@@ -197,3 +197,9 @@ export * from './view/styles/border.js';
 export * from './view/styles/margin.js';
 export * from './view/styles/padding.js';
 export * from './view/styles/utils.js';
+
+// Development / testing utils.
+export {
+	getData as _getModelData,
+	setData as _setModelData
+} from './dev-utils/model.js';
```

Comment:
As this is for unit tests I'd export also `stringify`, and `parse` from the same file, and the same exports from the `view.ts` file.